src/test/ui/proc-macro/visibility-path.stderr

```diff
@@ -0,0 +1,20 @@
+error: functions tagged with `#[proc_macro]` must be `pub`
+  --> $DIR/visibility-path.rs:12:1
+   |
+LL | / pub(self) fn outer(input: TokenStream) -> TokenStream {
```

Comment:
It would be better to span ob function name only, not entire its body.

Reply:
Done in https://github.com/rust-lang/rust/pull/70266.